README.md

```diff
@@ -34,29 +34,29 @@ getSessionCookie('username', 'password').then(getStatus);
 
 `setStatus` will set the status of the alarm system. It takes two parameters:
 
-* Session Cookie - This is retrieved using `getSessionCookie`
+* Access Token - This is retrieved using `getSessionCookie`
 * Mode - The mode can either be 'arm', 'home' or 'disarm'
 
 ```
-setStatus('sessionCookie', 'arm');
+setStatus('access_token', 'arm');
 ```
 
 As the example shows you will need to pass the sessionCookie into the setStatus method call, as this module is promise based you can simply chain the setStatus method call after your getSessionCookie call.
 
 ```
-getSessionCookie('username', 'password').then((sessionCookie) => {
-    setStatus(sessionCookie, 'arm');
+getSessionCookie('username', 'password').then((access_token) => {
```

Comment:
we should change this to getAccessToken, this is a breaking change but is worth making as it keeps it easier to maintain